addons/stock_picking_batch/__manifest__.py

```diff
@@ -27,11 +27,6 @@
     'demo': [
         'data/stock_picking_batch_demo.xml',
     ],
-    'assets': {
```

Comment:
Use a proper commit message.

Guidelines are always
[TAG] module_name: small description
[DEL] stock_picking_batch: remove choose line widget

Then complete the description.
Link to the commit that introduce the widget and explain why you can remove it. (In this case it's easy, just say that he was not use)